aerogear/mobile-security-service
# Aerogear 8470

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8470

## What
Breadcrumb and Buttons

## How
Using PF4 toolbar and some css hacks

## Verification Steps
 
1. Run 'make serve'
2. The page should look as below


<img width="970" alt="Screenshot 2019-03-13 at 18 05 59" src="https://user-images.githubusercontent.com/6498727/54303342-ae2e1500-45ba-11e9-8551-79f0e4ad87a0.png">
 

3. Click on the Home breadcrumb
4. The page should navigate to the Landing Page
5. Navigation history should also work

## Checklist:


## Progress

- [x] Rebase on [Table pr](https://github.com/aerogear/mobile-security-service/pull/91) once merged
- [x] Rebase on [get request pr](https://github.com/aerogear/mobile-security-service/pull/95) once merged
